superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberViz.tsx

```diff
@@ -62,6 +64,13 @@ type BigNumberVisProps = {
   trendLineData?: TimeSeriesDatum[];
   mainColor: string;
   echartOptions: EChartsCoreOption;
+  onContextMenu?: (
+    filters: QueryObjectFilterClause[],
+    offsetX: number,
```

Comment:
I'm seeing some strange tooltip positioning for Big Number and Big Number w/ Trendline:
<img width="311" alt="Big Number" src="https://user-images.githubusercontent.com/13007381/183691452-d262bc2f-4856-4ebc-9580-4913fec9c60e.png">

<img width="224" alt="Big Number w: Trendline" src="https://user-images.githubusercontent.com/13007381/183691605-3c3ea8e2-e7a0-4ea9-8d61-691b59a4a742.png">

Pie chart for reference:
<img width="438" alt="Pie" src="https://user-images.githubusercontent.com/13007381/183691815-926755eb-6e5b-4cd0-8b5e-88c2e75a0cfa.png">